UNITTESTS/CMakeLists.txt

```diff
@@ -19,6 +19,16 @@ endmacro()
 
 use_cxx14()
 
+
+if (MINGW)
```

Comment:
Is that the right way? I would rather make equeue work on Windows and Mac (for that matter), by enabling POSIX for these platforms in the platform abstraction layer of equeue.

Reply:
Good point.
Updated